### DISCUSSION TITLE
Why don't we get token address for any token we create?

### DISCUSSION THREAD

**Discussion by MD-SanjayKumar** [aptos-labs/aptos-developer-discussions#293] [Questions]
### Discord user ID

skite_in

### Describe your question in detail.

When we create aptos coin, why don't we get address for the created coin? Is CoinType enough in every aspect?

### Which operating system are you using?

Linux (Ubuntu, Fedora, Windows WSL, etc.)

### Which SDK or tool are you using? (if any)

Aptos CLI

**Comment by gregnazario** [comment]
## For Coin

Coins only need the coin type, which is used for inputs to functions e.g. `0x1::aptos_coin::AptosCoin`

More info here: https://aptos.dev/en/build/smart-contracts/aptos-coin

## For Fungible Asset
Fungible assets create addresses, so that will have an address in the output of the transaction.

More info here: https://aptos.dev/en/build/smart-contracts/fungible-asset
